sherpa-onnx/jni/jni.cc

```diff
@@ -1509,9 +1520,12 @@ JNIEXPORT void JNICALL Java_com_k2fsa_sherpa_onnx_SherpaOnnxOffline_delete(
 
 SHERPA_ONNX_EXTERN_C
 JNIEXPORT void JNICALL Java_com_k2fsa_sherpa_onnx_SherpaOnnx_reset(
-    JNIEnv *env, jobject /*obj*/, jlong ptr, jboolean recreate) {
+    JNIEnv *env, jobject /*obj*/, jlong ptr, jboolean recreate, jstring keywords) {
   auto model = reinterpret_cast<sherpa_onnx::SherpaOnnx *>(ptr);
-  model->Reset(recreate);
+  const char *p_keywords = env->GetStringUTFChars(keywords, nullptr);
+  std::string keywords_str = p_keywords;
+  model->Reset(recreate, keywords_str);
```

Comment:
```suggestion
  model->Reset(recreate, p_keywords);
```

You can pass `p_keywords` directly to the function.

Reply:
But `p_keywords` is `const char *` while the reset method accept `std::string`

Reply:
But `p_keywords` is `const char *` while the reset method accept `std::string`

Reply:
> But `p_keywords` is `const char *` while the reset method accept `std::string` ? It's incorrect type

Does it cause compiler errors? If yes, please post the error logs.

Reply:
you're right, it works normally